syntax.go

```diff
@@ -27,12 +27,16 @@ var (
 	reHead = regexp.MustCompile(`^(?:(.* \()([A-Z][a-z0-9:']+\.?(?:[- ][A-Za-z][a-z0-9:']*\.?)*)(\))|([A-Z][a-z0-9:']+\.?(?:[- ][A-Za-z][a-z0-9:']*\.?)*))(?:( \()([MF])(\)))?(?:( @ )([A-Z][a-z0-9:']*(?:[- ][A-Z][a-z0-9:']*)*))?( *)$`)
 	reMove = regexp.MustCompile(`^(-)( ([A-Z][a-z\']*(?:[- ][A-Za-z][a-z\']*)*)(?: \[([A-Z][a-z]+)\])?(?: / [A-Z][a-z\']*(?:[- ][A-Za-z][a-z\']*)*)* *)$`)
 	reStat = regexp.MustCompile(`^(\d+ HP)?( / )?(\d+ Atk)?( / )?(\d+ Def)?( / )?(\d+ SpA)?( / )?(\d+ SpD)?( / )?(\d+ Spe)?( *)$`)
+	reShiny = regexp.MustCompile(`^Shiny: *(\S*) *$`)
+	reNature = regexp.MustCompile(`^.+Nature *$`)
 
 	tmpl = template.Must(template.ParseFiles(filepath.Join(dir, "paste.tmpl")))
 )
 
-func renderPaste(w http.ResponseWriter, text, title, author, notes []byte) {
+func renderPaste(w http.ResponseWriter, text, title, author, notes []byte, openTeamsheet) {
 	sets := bytes.Split(text, []byte("\r\n\r\n"))
+	title = strings.ReplaceAll(title, "\n", "<br>")
```

Comment:
This will result in CRLF line endings translating to _two_ `<br>`s - not ideal. Title shouldn't ever be multiline anyway.

Reply:
wow... I very much meant for these to be notes lol